Makefile

```diff
@@ -73,17 +75,17 @@ plugin: regen-crd
 	@(cd $(PLUGIN_HOME); \
 		go vet ./... && \
 		go test -race ./... && \
-		GO111MODULE=on ${GOPATH}/bin/golangci-lint cache clean && \
-		GO111MODULE=on ${GOPATH}/bin/golangci-lint run --timeout=5m --config ../.golangci.yml)
+		GO111MODULE=on golangci-lint cache clean && \
+		GO111MODULE=on golangci-lint run --timeout=5m --config ../.golangci.yml)
```

Comment:
this assumes `golangci-lint` is in PATH - this may not be the case since GOPATH might not be in PATH

Reply:
in this makefile we have it like this in other places, like in `getdeps`, I think assuming it's in the PATH should be fine, as long as $GOPATH is in the PATH hahaha